modules/rocketmq/pom.xml

```diff
@@ -44,23 +44,14 @@
 
         <dependency>
             <groupId>org.apache.rocketmq</groupId>
-            <artifactId>rocketmq-client</artifactId>
+            <artifactId>rocketmq-broker</artifactId>
```

Comment:
I have checked your broker dependency is just for test. Could you scope this dependency to test ?